docs/CONTRIBUTING.md

```diff
@@ -66,6 +80,20 @@ After you are able to successfully build JsDbg, please follow the steps below to
 
 8. The JsDbg server will launch and prompt you to select a browser. If the debugger is attached to one browser, it is recommended that you choose a different browser for JsDbg.
 
+#### GDB
+
+After following the build instructions above, type `jsdbg` at the gdb prompt to run it. It will tell you the URL to open in your browser.
```

Comment:
I think we should clarify that the "build instructions" above are the Linux build instructions.

Reply:
Done